backend/app/Console/Commands/GCSCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Google\Cloud\Storage\StorageClient;
+use Illuminate\Support\Carbon;
+
+class GCSCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'gcs:backup';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'GCSへバックアップデータを送るコマンド';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $client = new StorageClient();
+        $bucket = $client->bucket(env('GCS_PACKET')); // 作成したバケット
+
+
+        // ディレクトリハンドルの取得
+        $dirH = opendir(storage_path('app/laravel-backup/'));
+        while (false !== ($fileList[] = readdir($dirH)));
+        closedir($dirH);
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter #1 $dir_handle of function closedir expects resource|null, resource|false given.